Aztec/Classes/Public/Extensions/NSRange+Helpers.swift

```diff
@@ -34,6 +34,12 @@ extension NSRange
         }
     }
 
+    /// Returns the maximum Location.
+    ///
+    var lastLocation: Int {
```

Comment:
This is fairly minor: but I'd replace `last` with `end` since the term is used in other places in our code.

It's also more frequently used in Apple's code.